eng/pipelines/templates/jobs/archetype-sdk-integration.yml

```diff
@@ -34,6 +34,30 @@ parameters:
       OSVmImage: "macOS-10.15"
       TestType: "browser"
       CloudType: public
+    MaxVersion_Node (Public):
```

Comment:
Because of the way JS is currently set up you'll also have to add this to every `tests.yml` file because each defines its own matrix. If the intent is to always have this present regardless of the original test matrix, you could do something like: 

```
      matrix:
        ${{ each matrixEntry in parameters.Matrix }}:
          ${{ matrixEntry.key }}:
            # TestResultsFiles
            ${{ if eq(matrixEntry.value.TestType, 'browser' )}}:
              TestResultsFiles: "**/test-results.browser.xml"
            ${{ if eq(matrixEntry.value.TestType, 'node') }}:
              TestResultsFiles: "**/test-results.xml"

            # Remaining entries from the matrix object
            ${{ insert }}: ${{ matrixEntry.value }}

        # Include these entries in all test matrices. Note that TestResultsFile
        # is specified because the template logic above is not running in this
        # context.
        MaxVersion_Node (Public):
          OSVmImage: "ubuntu-16.04"
          TestType: "node"
          CloudType: public
          PublishCodeCoverage: false
          DependencyVersion: max
          TestResultsFiles: "**/test-results.xml"
        MaxVersion_Browser (Public):
          OSVmImage: "ubuntu-16.04"
          TestType: "browser"
          CloudType: public
          PublishCodeCoverage: false
          DependencyVersion: max
          TestResultsFiles: "**/test-results.browser.xml"
        MinVersion_Node (Public):
          OSVmImage: "ubuntu-16.04"
          TestType: "node"
          CloudType: public
          PublishCodeCoverage: false
          DependencyVersion: min
          TestResultsFiles: "**/test-results.xml"
        MinVersion_Browser (Public):
          OSVmImage: "ubuntu-16.04"
          TestType: "browser"
          CloudType: public
          PublishCodeCoverage: false
          DependencyVersion: min
          TestResultsFiles: "**/test-results.browser.xml"
```

Reply:
This is great